WarehouseManagement.Core/DTOs/Marker/MarkerVendorDto.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace WarehouseManagement.Core.DTOs.Marker;
```

Comment:
are these (MarkerVendorDto and MarkerZoneDto) for the relations between a `Marker` and `Zone/Vendor`?

Reply:
Yes, the idea was, when you send request to the GetMarker/AllMarkers endpoint , to return the Marker id/name plus information about all the connected Vendors/Deliveries/Zones to it.